IPython/utils/process.py

```diff
@@ -41,6 +41,9 @@ def find_cmd(cmd):
         from IPython.utils.process import pycmd2argv
         argv = pycmd2argv(get_ipython_module_path('IPython.terminal.ipapp'))
 
+    Note, The code for pycmd2argv has been removed now as it was not used
+    anywhere. get_ipython_module_path should give us that same result.
```

Comment:
If we're removing this function,  we should fix the example not to use it, rather than adding a note that the example is outdated.

The modern advice would be to launch IPython using Python's ability to launch a module:

```
argv = [sys.executable, '-m', 'IPython']
```